docs/manuals/scripts/generate-supported-versions-timeline.py

```
@@ -0,0 +1,486 @@
+        )
+        if current_segment:
+            current_x = x_position(current_segment[0], window_start, window_end)
+            current_width = (
+                x_position(current_segment[1], window_start, window_end) - current_x
+            )
+            parts.append(
+                f'      <rect class="{current_class}" x="{current_x}" y="{row_y}" '
+                f'width="{current_width}" height="{BAR_HEIGHT}" rx="4"/>\n'
+            )
+            text_class = "small-label" if release.projected else "label"
+            text_value = "current" if release.projected else "current"
```

Ruff (0.15.15)

[warning] Useless `if`-`else` condition

(RUF034)